sdk/servicebus/azure-servicebus/tests/mgmt_tests/async/test_mgmt_queues_async.py

```diff
@@ -413,10 +414,11 @@ async def test_async_mgmt_queue_create_duplicate(self, servicebus_namespace_conn
         finally:
             await mgmt_service.delete_queue(queue_name)
 
-    @CachedResourceGroupPreparer(name_prefix='servicebustest')
-    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
-    async def test_async_mgmt_queue_update_success(self, servicebus_namespace_connection_string, servicebus_namespace, **kwargs):
-        mgmt_service = ServiceBusAdministrationClient.from_connection_string(servicebus_namespace_connection_string)
+    @ServiceBusPreparer()
+    @pytest.mark.live_only
```

Comment:
This typo seems to be the reason this test is failing in pipelines -- it's just not getting skipped properly. It passes in live mode for me locally 🙂
```suggestion
    @pytest.mark.live_test_only
```